.github/workflows/clippy.yml

```
@@ -95,6 +95,5 @@ jobs:
     # Cleanup
     - name: Run cargo-cache --autoclean
       run: |
-        cargo +nightly install cargo-cache --debug
-        find ~/.cargo/bin ! -type d -exec strip {} \;
-        cargo cache --autoclean
+        cargo install --force cargo-cache --no-default-features --features ci-autoclean cargo-cache 
```

```suggestion
        cargo +nightly install cargo-cache --no-default-features --features ci-autoclean cargo-cache 
```

The nightly version of cargo updates the package if it is not installed. `--force` builds it every time and won't use the cache. 

I think we should keep the `find ~/.cargo/bin ! -type d -exec strip {} \;` line for smaller cache sizes.

---

Mh stripping reduces size from 984K to 208K on my system, alright.

---

Oh I never tested that. Ok that isn't really relevant, so we can remove it, I just copied it from travis.

---

It was probably relevant because we installed cargo-cache with --debug perviously

---

:man_shrugging: I don't really care if we keep it or delete it. The caches 80-100MB big anyway, so 1MB+/- doesn't make a difference.